stress-test.yaml

```diff
@@ -0,0 +1,214 @@
+      params:
+        <<: *infrastructure_vars
+        <<: *deploy_infrastructure_vars
+    on_failure:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Infrastructure Deploy Failed
+            color: danger
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+    on_success:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Infrastructure Deploy Passed
+            color: good
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+
+##############################################################################
+# Run Stress Test
+##############################################################################
+
+  - name: run-stress-test
+    serial_groups: [run-stress-test]
+    plan:
+    - get: eq-survey-runner-benchmark
+    - task: Run Survey Runner Stress Test
+      file: 'eq-survey-runner-benchmark/ci/run-locust.yaml'
+      params:
+        <<: *run_locust_vars
+      on_failure:
+        put: slack-alert
+        params:
+          channel: '#((slack_channel_name))'
+          attachments:
+            - pretext: Survey Runner Stress Test Failed
+              color: danger
+              title: Concourse Build $BUILD_ID
+              title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+      on_success:
+        put: slack-alert
+        params:
+          channel: '#((slack_channel_name))'
+          attachments:
+            - pretext: Survey Runner Stress Test Passed
+              color: good
+              title: Concourse Build $BUILD_ID
+              title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+
+
+##############################################################################
+# Destroy Infrastructure
+##############################################################################
+
+  - name: destroy-infrastructure
+    serial_groups: [run-stress-test]
+    plan:
+    - get: eq-terraform-load-generator
+    - get: eq-survey-runner-benchmark
+      passed: [build-survey-runner-benchmark, run-stress-test]
+    - task: Destroy Infrastructure
+      file: 'eq-terraform-load-generator/ci/destroy_infrastructure.yaml'
+      params:
+        <<: *infrastructure_vars
+        <<: *destroy_infrastructure_vars
+    on_failure:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Infrastructure Deploy Failed
+            color: danger
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+    on_success:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Infrastructure Deploy Passed
```

Comment:
These Slack alerts should be `Destroy` rather than `Deploy`. I can see that this was probably copied from the existing pipeline, so that will need to be changed as well.